Source/Versions.swift

```diff
@@ -14,6 +14,19 @@ public enum Semantic {
   case Major, Minor, Patch, Same, Unknown
 }
 
+public class App {
+
+  static var version: String = {
+    var version: String = ""
```

Comment:
I think string is ok, because you have to specify version when releasing an app.